src/main/java/carracing/domain/Winner.java

```diff
@@ -12,16 +14,16 @@ public Winner(List<Car> cars) {
         this.cars = cars;
```

Comment:
우승자 객체도 크게 본다면 Car의 일급 컬렉션으로 볼수 있겠네요 ! 
Cars 와 Winner 객체를 합쳐도 되지 않을까요 ? 
혹시 일급컬렉션에 대하여 처음 듣거나 알지못한다면 아래 문서를 참고해주세요 :) 
https://jojoldu.tistory.com/412
```suggestion
        this.cars = Collections.unmodifiableList(cars);
```

Reply:
외부에서 Cars 내부에 있는 List<Car>을 수정하지 않으려는 의도이면 
Collections.unmodifiableList(cars); 이 기능으로는 내부객체의 수정을 막을 수 없습니다.!
`@Test
    void defensive_copy_테스트() {
        List<Car> carList = new ArrayList<>();
        Cars carsList = new Cars(carList);

        assertThat(carList.size()).isEqualTo(0);
        assertThat(carsList.getCars().size()).isEqualTo(0);

        car = new Car("car1");
        car.setPosition(new CarPowerCondition());
        carList.add(car);

        assertThat(carsList.getCars().size()).isEqualTo(0); // fail

    }`